modules/library/main/src/main/java/org/geotools/styling/visitor/RescalingMode.java

```diff
@@ -150,5 +141,17 @@ public Expression rescaleToExpression(Expression scaleFactor, Measure measure) {
                     Measure.ff.literal(measure.uom), scaleFactor, Measure.ff.literal(this));
         }
     }
+    
+    /**
+     * Translates between units and their shortcuts (we can only get the full name from the unit
+     * object
+     */
+    final Map<Unit, String> UNIT_SYMBOLS = new HashMap<Unit, String>() {
+        {
+            put(NonSI.PIXEL, "px");
+            put(NonSI.FOOT, "ft");
+            put(SI.METER, "m");
+        }
+    };
```

Comment:
It's not obvious: `UNIT_SYMBOLS` was previously a field of the `KeepUnits` enum constant only. Now it is a field of the `RescalingMode` enum itself, making it accessible for both, `KeepUnits` and `Pixels`.